app/templates/sites/network-sites/network-site/buildings/new.hbs

```diff
@@ -0,0 +1,24 @@
+<h2>New Building</h2>
+
+{{#g-maps-container
+    lat=model.site.lat
+    lng=model.site.lng
+    zoom=16
+    buildingMarkers=buildingMarkers
```

Comment:
`g-maps-container` has a method that creates markers from models, but it only works for `networkSite` models. So I'm passing in markers through `buildingMarkers` instead. Thoughts?